SDK Libraries/receiver-ssf-api.yaml

```diff
@@ -0,0 +1,240 @@
+openapi: 3.0.3
+info:
+  version: 1.0.0-oas3
+  title: CAEP SSF API
+  description: CAEP.dev Receiver SSF API
+  contact:
+    name: API Support
+    email: hello@caep.dev
+  license:
+    name: MIT
+    url: https://developer.sgnl.ai/mit-license
+  termsOfService: https://sgnl.ai/TERMS-OF-SERVICE
+servers:
+  - url: https://caep-ssf.sbx.sgnl.host/ssf
```

Comment:
Do we need to specify this?

Reply:
If users want to query on our Public SSF then yes. But I'll switch the URL to the prod one.